mitxgraders/helpers/mitmath/specify_domain.py

```diff
@@ -95,12 +107,33 @@ def make_shape_validator(shape):
     >>> validate_vec4('cat')
     Traceback (most recent call last):
     Invalid: received a str, expected a vector of length 4
+
+    Instead of specifying a tuple shape, you can speicfy 'square' to demand
```

Comment:
specify

Reply:
After I got my spell-chcker in atom, I switched linters, and it warned me near the beginning of these strings :(